src/sentry/web/api.py

```diff
@@ -306,6 +307,9 @@ def post(self, request, **kwargs):
             # bubble up as an APIError.
             data = None
 
+        if getattr(settings, 'ENABLE_PUBSUB', False):
```

Comment:
I'd run this once at module level, and use that as well to even conditionally import `pubsub`. That avoids needing to re-evaluate this every time as well since it's an immutable value.